CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 # Release History
+## 0.11.0 (2022-09-28)
+
+### Behavior Changes
+- When adding a package using `session.add_packages()` with a specific version to UDFs or stored procedures, if the version is not supported by Snowflake, an exception will be raised immediately instead of when registering the UDF or stored procedure.
```

Comment:
I feel this can be more specific with something like below. The break change is on this API, not on registering a UDF/SP.
`Session.add_packages()` now raises `<the error class>` when the version of a package can't be found in Snowflake Anaconda channel.
What do you think @sfc-gh-mabrennan ?

Reply:
I also want to add the previous behavior as well. How about:

```
- `Session.add_packages()` now raises `ValueError` instead of `SnowparkSQLException` when the version of a package cannot be found in Snowflake Anaconda channel. 
```

Reply:
`Session.add_packages()` didn't raise SnowparkSQLException before. Only when you create a udf/sproc, then SnowparkSQLException will be raised

Reply:
I think what you have written is correct. To be clear, you can re-word the second line as "Previously, error was raised later in the udf/sp registration step instead of `add_packages` step."

Reply:
@sfc-gh-jdu sorry I edited it as I also realized what I previously wrote was wrong...This is the current version:

```
- `Session.add_packages()` now raises `ValueError` when the version of a package cannot be found in Snowflake Anaconda channel. Previously, `Session.add_packages()` succeeded, and a `SnowparkSQLException` exception was raised later in the UDF/SP registration step.
```

Reply:
perfect, thanks!